frontend/src/metabase/xray/components/Insights.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { InsightCard } from "metabase/xray/components/InsightCard";
+
+export class Insights extends Component {
+    props: {
+        features: any,
+    }
+
+    render() {
+        const { features } = this.props;
+
+        const parametrizedInsights = Object.entries(features["insights"])
+            // temporary hacks as we have two formats
```

Comment:
Can we elaborate on this? Examples of the formats maybe or some reason about why there are two?

Reply:
whoops. that's an old hack which can be safely removed

Reply:
(that was from era when we were trying to decide whether the backend provides complete sentences or just the computed values, we decided to go with the latter approach)

Reply:
removed